src/main/webapp/web.xml

```diff
@@ -45,7 +63,7 @@
 
     <servlet>
         <servlet-name>allCars</servlet-name>
-        <servlet-class>mate.controller.car.GetAllCarsController</servlet-class>
+        <servlet-class>mate.controller.driver.GetMyCurrentCarsController</servlet-class>
```

Comment:
We don't need to delete GetAllCarsController, we must still have opportunity to get all cars